src/stdune/path.ml

```diff
@@ -779,12 +858,23 @@ let is_descendant t ~of_ =
 
 let append_relative a b =
```

Comment:
This function should be unused if we make the `Relative` module unused, and given its weird nature (same issue as above: ".." results in error) maybe we should remove it.